sdk/communication/azure-communication-chat/README.md

```diff
@@ -293,7 +293,7 @@ Use `update_topic` method to update a thread's properties. `topic` is used to de
 topic = "new topic"
 chat_thread_client.update_topic(topic=topic)
 
-chat_thread = chat_client.get_properties(thread_id)
+chat_thread = chat_thread_client.get_properties(thread_id)
 
 assert chat_thread.topic == topic
```

Comment:
We don't need to pass `thread_id` here. You can refer to this [sample](https://github.com/Azure/azure-sdk-for-python/blob/master/sdk/communication/azure-communication-chat/samples/chat_thread_client_sample.py#L83-L97)

```suggestion
chat_thread_properties = chat_thread_client.get_properties()

assert chat_thread_properties .topic == topic
```